contrib/refmt

```diff
@@ -58,7 +42,7 @@
 #                                            'delay': '0.000640848',
 #                                            'lastCheck': '0.0',
 #                                            'valid': True,
-#                                            'version': 3}}
+#
```

Comment:
Last line deleted by mistake?

Reply:
Not recovered completely by mistake. But yes, I'll fix.